app/enterprise/2.5.x/external-plugins.md

```diff
@@ -577,6 +577,56 @@ hardware and to avoid confusion.
 
 <center><img title="Latency" src="/assets/images/docs/external-plugins/latency.png"/></center>
 
+## Use external plugins in container and Kubernetes
+
+When using external plugin servers in container, they need to be installed inside the container
+alongside with any external plugins as well.
+
+For Golang, user may build external plugins in embedded server mode in a builder container
```

Comment:
I think "user may" makes this confusing because it implies there are are other options.

```suggestion
For Golang, build external plugins in embedded server mode in a builder container
```

Reply:
When you say, "For Golang" what exactly do you mean? For instances when you are building external plugins written in Golang? 
```suggestion
For plugins written in Golang, build external plugins in embedded server mode in a builder container
```